recipes/xcube/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "xcube" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+source:
+  url: "https://github.com/dcs4cop/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 35a934429ff173b383203395ad4ad513c405a15bd5a6c2a319d4d8fa6816f116
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - xcube = xcube.cli.cli:main
+    - xcube-gen = xcube.cli.gen:main
+    - xcube-restime = xcube.cli.restime:main
+requirements:
+  host:
+    # Python
+    - python >=3.6
+    - libblas
+    - libcblas
```

Comment:
These 2 are unneeded.

Reply:
@ocefpaf, @xhochy, this needs to be removed before merge

Reply:
Removed in the next push. They were required by the blas dependency which we removed